journalbeat/hooks/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+exec 2>&1
+
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:{{pkgPathFor "core/systemd"}}/lib
```

Comment:
This shouldn't be necessary.  The Habitat build process will embed the correct rpath information in the binaries produced:  `Library rpath: [/hab/pkgs/core/glibc/2.27/20190115002733/lib:/hab/pkgs/core/systemd/239/20190117202145/lib` .  As a general rule, we try to avoid LD_LIBRARY_PATH with Habitat as it will always load from that path before any of the embedded rpath paths.  In this case there would be no effect, however if it were to include a system path you would end up with unexpected behavior.

Reply:
Without this line I get errors that it cannot open the systemd library. Using strace I saw that it was looking in the glibc path. I tried using patchelf to get it in the rpath, but ended up with it segfaulting. If there's something I'm missing to get it working let me know and I'll update it.

Reply:
Interesting, I wasn't seeing that.  Let me dig a little deeper and see what I can uncover.

Reply:
We did some further testing and were unable to recreate the behavior I saw before. Pushed a change removing the LD_LIBARY_PATH line.